WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewHolder.kt

```diff
@@ -81,8 +83,17 @@ sealed class PrepublishingHomeViewHolder(
                 storyTitle.setSelection(title.text.length)
             }
 
+            val focusStoryTitleEditTextAndShowKeyboard = {
+                storyTitle.focusAndShowKeyboard()
+                storyTitle.postDelayed({ storyTitle.requestFocus() }, STORY_TITLE_EDIT_TEXT_REQUEST_FOCUS_DELAY)
+            }
+
+            storyTitleLayout.setOnClickListener {
+                focusStoryTitleEditTextAndShowKeyboard()
+            }
+
             storyTitle.postDelayed({
-                storyTitle.requestFocus()
+                focusStoryTitleEditTextAndShowKeyboard()
             }, STORY_TITLE_EDIT_TEXT_REQUEST_FOCUS_DELAY)
```

Comment:
There's now a chained delay of `STORY_TITLE_EDIT_TEXT_REQUEST_FOCUS_DELAY` here: 
1. to call focusStoryTitleEditTextAndShowKeyboard, at this point the focus and keyboard are requested
2. another one to requestFocus (that acttually has the effect of getting the keyboad up).

This is weird in reality because tapping on the NEXT button on the Story composer makes a sequence of weird movements instead of everything happening all at once (or at least closely together). Also, `STORY_TITLE_EDIT_TEXT_REQUEST_FOCUS_DELAY` is 1000 millis, probably 2 seconds is too long. 

Here's a video showing this https://cloudup.com/c-wQOeSvHE1

I'm going to check this in a deeper way and see what I can find

Reply:
Thanks! I didn't see that delay initially. I will check this out as well to see.